app/controllers/concerns/foreman/controller/parameters/user_column.rb

```diff
@@ -0,0 +1,15 @@
+module Foreman::Controller::Parameters::UserColumn
```

Comment:
there is no need for strong param filtering if you don't use mass assignment of the values.